build_rootfs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+rm -fv *.tar.xz
+cp crystal/pacman.conf crystal/airootfs/etc/.
+cp chrooted-bootstrap.sh chrooted.sh
+
+if [[ "$1" == "" ]]; then
+    WORKDIR=$(mktemp -d)
+else
+    WORKDIR="$1"
+    if [[ ! -d "$WORKDIR" ]]; then
+        mkdir -p "$WORKDIR"
+    fi
+fi
+
+echo "Built on $(date +"%D @ %T EST")" > crystal/airootfs/etc/buildstamp
+
+time sudo ./mkarchiso -m bootstrap -v -w $WORKDIR -o . crystal
```

Comment:
this is very similar to the normal build.sh, maybe make it generate a rootfs if an envvar is set (e.g. CRYSTAL_GEN_ROOTFS)

Reply:
I believe having seperate scripts or adding a flag instead of an envvar is the better approach, since that's easier to use and discover than an envvar